src/sentry/static/sentry/app/views/eventsV2/eventDetails/index.tsx

```diff
@@ -57,20 +56,17 @@ class EventDetails extends React.Component<Props> {
 
     return (
       <SentryDocumentTitle title={documentTitle} objSlug={organization.slug}>
-        <React.Fragment>
-          <GlobalSelectionHeader organization={organization} />
```

Comment:
I wonder if the page shifting up/down when going from results to the details view and back to the results will be distracting.

Reply:
@markstory the page shift didn't seem to be an issue at all on local dev; switching pages on Discover shows a loading state in between route changes. Especially since local dev is as fast as data can be loaded vs prod.

Reply:
@doralchan thoughts on this potential issue?